### ISSUE TITLE
[General Issue]只训练了一个epoch就自动停止了，也没报错

### ISSUE THREAD

**Issue by 19961115** [PaddlePaddle/PaddleSeg#1838]
Thanks for your issue. To help us solve the issue better, please provide following information：
 1. PaddleSeg version: (please specify the branch as well，e.g. PaddleSeg release/2.3)
 2. PaddlePaddle version: (e.g. PaddlePaddle 2.1.0)
 3. Operation system: (e.g. Linux/Windows/MacOS)
 4. Python version: (e.g. Python3.6/7/8)
 5. CUDA/cuDNN version: (e.g. CUDA10.2/cuDNN 7.6.5)
 6. Additional context: (Add any other context about the problem)

---

欢迎您反馈PaddleSeg使用问题，辛苦您提供以下信息，方便我们快速定位和解决问题：
 1. PaddleSeg版本：（请提供版本号和分支信息，如PaddleSeg release/2.3）
 2. PaddlePaddle版本：（如PaddlePaddle 2.1.0）
 3. 操作系统信息：（如Linux/Windows/MacOS）
 4. Python版本号：（如Python3.6/7/8）
 5. CUDA/cuDNN版本：（ 如CUDA10.2/cuDNN 7.6.5等）
 6. 其他内容: （增加其他与问题相关的内容）

**Comment by juncaipeng** [comment]
麻烦将配置的yaml文件发出来看下。

**Comment by lyf6** [comment]
i have also met this problem.
PaddleSeg版本：PaddleSeg release/2.5.0
PaddlePaddle版本：PaddlePaddle 2.3.0
操作系统信息：Windows
Python版本号：Python3.6
CUDA/cuDNN版本：CUDA10.2/cuDNN 7.6
![飞浆](https://user-images.githubusercontent.com/17488327/177457633-8ec93940-1634-4ab8-8b61-203de2c268b4.png)
here is my config file:
batch_size: 2
iters: 2000
loss:
  coef:
  - 1
  - 1
  - 1
  types:
  - min_kept: 260000
    type: OhemCrossEntropyLoss
  - min_kept: 260000
    type: OhemCrossEntropyLoss
  - min_kept: 260000
    type: OhemCrossEntropyLoss
lr_scheduler:
  end_lr: 0
  learning_rate: 0.001
  power: 0.9
  type: PolynomialDecay
  warmup_iters: 1000
  warmup_start_lr: 1.0e-05
model:
  arm_out_chs:
  - 32
  - 64
  - 128
  backbone:
    pretrained: https://bj.bcebos.com/paddleseg/dygraph/PP_STDCNet1.tar.gz
    type: STDC1
  seg_head_inter_chs:
  - 32
  - 64
  - 64
  type: PPLiteSeg
optimizer:
  momentum: 0.9
  type: sgd
  weight_decay: 0.0005
train_dataset:
  dataset_root: C:/Users/Administrator/Documents/ArcGIS/workdir
  ignore_index: 255
  mode: train
  num_classes: 2
  train_path: C:/Users/Administrator/Documents/ArcGIS/workdir\train.txt
  transforms:
  - max_scale_factor: 1.2
    min_scale_factor: 0.8
    type: ResizeStepScaling
  - crop_size:
    - 512
    - 512
    type: RandomPaddingCrop
  - type: RandomHorizontalFlip
  - type: RandomVerticalFlip
  - brightness_prob: 0.5
    brightness_range: 0.3
    contrast_prob: 0.5
    contrast_range: 0.3
    hue_prob: 0.5
    hue_range: 15
    saturation_prob: 0.5
    saturation_range: 0.3
    type: RandomDistort
  - type: Normalize
  type: Dataset
val_dataset:
  dataset_root: C:/Users/Administrator/Documents/ArcGIS/workdir
  mode: val
  num_classes: 2
  transforms:
  - type: Normalize
  type: Dataset
  val_path: C:/Users/Administrator/Documents/ArcGIS/workdir\train.txt

**Comment by lyf6** [comment]
I guess there is something wrong with the dataloader.
